src/vs/workbench/contrib/chat/electron-sandbox/actions/voiceChatActions.ts

```diff
@@ -263,7 +265,9 @@ class VoiceChatSessions {
 		const speechToTextSession = session.disposables.add(this.speechService.createSpeechToTextSession(cts.token));
 
 		let inputValue = controller.getInput();
-		const acceptTranscriptionScheduler = session.disposables.add(new RunOnceScheduler(() => session.controller.acceptInput(), 1200));
+
+		const delay = this.configurationService.getValue<number>('accessibility.voiceChat.listeningDuration');
```

Comment:
You need to validate this is a `number`. The user can put in any value.

Reply:
Oh okay, I thought it was handled by TypeScript but I will add that verification.

Reply:
After testing I am unable to add any character other than a number in the settings page, do you still want me to put in the check?
![image](https://github.com/microsoft/vscode/assets/84996242/4deee6bb-654d-474b-974b-2cd664460b2e)

Reply:
Users can edit the settings JSON file directly.